README.md

```diff
@@ -5,7 +5,15 @@ LittlevGL ported to STM32F746G-DISCO
 1. Visit https://atollic.com/resources/download/ to download and install the Atollic IDE.
 2. Clone (or download) this GitHub repository to a folder on your computer and update the submodules:
 
+```sh
 git clone --recursive https://github.com/littlevgl/stm32f746_disco_no_os_atollic.git
+```
 
 3. Import the project into your workspace.
 4. Build and run!
+
+# How to update/pull repo
+
+```sh
+git submodule update --init
```

Comment:
@seyyah Don't we need `git pull` here as well?

Reply:
I also use `git pull` to update submodules. Does `git submodule update` really work once the submodule is already there?

Reply:
@seyyah See https://stackoverflow.com/a/5828396.

Reply:
You may be right. I'll try ASAP.

Reply:
You're right @kisvegabor @embeddedt